src/Resources/PaymentTypes/OpenBanking.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace UnzerSDK\Resources\PaymentTypes;
+
+use UnzerSDK\Traits\CanAuthorize;
+use UnzerSDK\Traits\CanDirectCharge;
```

Comment:
I would remove these traits, since they are redundant. And a would not add them for new payment methods, because they are limited in transaction properties they can handle.